### DISCUSSION TITLE
Could you tell me where to find an official pockethost docker image please ?

### DISCUSSION THREAD

**Discussion by rodridev97** [pockethost/pockethost#382] [Q&A]
Hello, I find it difficult how to set up local pockethost on my machine, the official documentation does not help much or is not very explanatory. If you could tell me if there is an official pockethost docker image or a step-by-step guide on how to set up local pockethost, I would appreciate it.

**Comment by benallfree** [comment]
Running locally is not supported just yet, the CLI is still in development. Please join https://discord.gg/nVTxCMEcGT for the most current information.
